README.md

```diff
@@ -1,4 +1,35 @@
 # xfero
-File Transfer Orchestration software
+A practical File Transfer Orchestration solution for business
 
-This needs filling in
+## Why xfero?
+**THE MISSING LINK IN BUSINESS FILE TRANSFER SERVICES**
+
+Xfero is the complete orchestration framework for business file transfer services with a mission to simplify complex file transfer integration while giving business users the ability to manage configuration rather than relying on techies.
```

Comment:
Xfero is a complete orchestration framework that simplifies complex file transfer integration for business file transfer services whilst giving the business community the ability to manage these services without having to engage with their technical department.